api/envoy/api/v2/eds.proto

```diff
@@ -80,6 +81,11 @@ message ClusterLoadAssignment {
     //    "lb"_drop = 20%  // 50% of the remaining 'actual' load, which is 40%.
     //    actual_outgoing_load = 20% // remaining after applying all categories.
     repeated DropOverload drop_overloads = 2;
+
+    // Priority levels and localities are considered overprovisioned with this factor. This means
+    // that we don't consider a priority level or locality unhealthy until the percentage of healthy
+    // hosts multiplied by kOverProvisioningFactor drops below 100.
```

Comment:
It looks like `kOverProvisioningFactor` isn't defined anywhere?

Reply:
done